Stacks_Using_Linked_Lists/Stacks.c

```diff
@@ -27,13 +29,52 @@ void POP(struct stack *s)
     if(s->top == -1)
         printf("\nThe stack is empty");
     else
-        {
-            int value;
-            value = s->data[s->top];
-            printf("\nThe poped value is %d",value);
-            s->top--;
-        }
+    {
+        int value;
+        value = s->data[s->top];
+        printf("\nThe poped value is %d",value);
+        s->top--;
+    }
+    count--;
 }
+
+/* function to check if the stack is empty or not */
+int isEmpty() 
+{ 
+    return s->top == NULL; 
+} 
+  
+/* function to return top element in a stack */ 
+int peek() 
+{ 
+    //check for empty stack 
+    if (!isEmpty(s->top)) 
+        return s->data[s->top]; 
+    else
+        exit(EXIT_FAILURE); 
```

Comment:
error: 'EXIT_FAILURE' undeclared (first use in this function)

Reply:
EXIT_FAILURE macros is to indicate program execution status.It indicates an unsuccessful execution of a program

Reply:
In which header file is this defined? I am getting error.